src/components/avatar/Avatar.js

```diff
@@ -1,20 +1,23 @@
-import React, { Component } from 'react';
+import React from 'react';
 import './Avatar.scss';
 import Image from 'react-bootstrap/Image';
 
 import defaultAvatar from '../../assets/images/avatar-default.webp';
 
-class Avatar extends Component {
-
-    render() {
-
-        const imgSrc = this.props.src || defaultAvatar;
-
+const Avatar = ({imgSrc=defaultAvatar, avatarSize="small"}) => {
```

Comment:
How about making this change only on the `className`?

```suggestion
const Avatar = ({imgSrc=defaultAvatar, avatarSize="small"}) => {
    return (
            <div className={avatarSize === "small" ? "smallAvatarContainer" : "largeAvatarContainer"}>
                <Image alt="userImage" src={imgSrc} roundedCircle />
            </div>
        )
```

Reply:
Done